packages/comms/lib/src/message_sink_register.dart

```diff
@@ -1,5 +1,7 @@
 part of '../comms.dart';
 
+typedef Log = void Function(String message);
```

Comment:
```suggestion
typedef _LoggerCallback = void Function(String message);
```

Other packages name it like this ([1](https://github.com/leancodepl/patrol/blob/master/packages/patrol/lib/src/native/native_automator.dart#L10) [2](https://github.com/leancodepl/flutter_corelibrary/blob/master/packages/login_client/lib/src/login_client.dart#L28))